i18n/en-US.properties

```diff
@@ -808,6 +808,8 @@ boxui.presence.timeSinceLastPreviewed = Previewed {timeAgo}
 boxui.queryBar.addFilterButtonText = + Add Filter
 # Text on the apply filter button, on click applies the filters
 boxui.queryBar.applyFiltersButtonText = Apply
+# Text on the columns button, if one column has been hidden then it will display this text
+boxui.queryBar.columnHiddenButtonText = 1 Column Hidden
```

Comment:
We don't need multiple messages... make use of `react-intl`. The unified message could look like:

`{count, plural, =1 {Column} other {Columns} Hidden`